examples/fcu_arm_test.cpp

```diff
@@ -25,37 +25,54 @@ int main(int argc, char *argv[]) {
     const size_t baudrate = (argc>2) ? std::stoul(argv[2]) : 115200;
 
     std::chrono::high_resolution_clock::time_point start, end;
-    bool feature_changed = false;
-start:
-    fcu::FlightController fcu(device, baudrate);
+    //bool feature_changed = false;
+//start:
+    std::cout << "making FC" << std::endl;
+    fcu::FlightController fcu;
 
     // wait until connection is established
     // get unique box IDs
     start = std::chrono::high_resolution_clock::now();
-    fcu.initialise();
+    fcu.connect(device, baudrate);
     end = std::chrono::high_resolution_clock::now();
     std::cout<<"ready after: "<<std::chrono::duration_cast<std::chrono::milliseconds>(end-start).count()<<" ms"<<std::endl;
-
+/*
     // on cleanflight, we need to enable the "RX_MSP" feature
     if(fcu.isFirmwareCleanflight()) {
```

Comment:
INAV has MSP enabled by default. Which reminds me, there will need to be some validation of the feature list on a per-firmware basis. I haven't changed the list because it's not important for INAV operation.

Reply:
At least cleanflight and betaflight have `RX_MSP` disabled by default because they assume that you are going to use a proper "physical" remote control. This makes sense since only a few people actually want to control their drone via MSP. However, this function is just for convenience, to enable control via MSP without needing to start the GUI and find the option.

Reply:
The main reason I disable this code block is because it results in a reboot of the FC. That makes the /dev/ttyACM0 device disappear and the library doesn't really handle that.

Reply:
There was a sleep to wait until the FC rebooted. It used to work as long as the device path did not change in between.
This did not work with your FC? Would it work if you increase the sleep time from 5 seconds to e.g. 10?

Reply:
I think this would actually take some development to do properly in terns of stopping the serial device and reconnecting it.

Reply:
Ok, then just remove this commented code.
I will look later into this.